src/main/java/org/broadinstitute/hellbender/tools/walkers/sv/EvaluateConcordanceOnNonGenotypedSVVCFs.java

```diff
@@ -0,0 +1,497 @@
+                "and has been under active developments. " +
+                "This tool pads the interval around the interval bounded by POS and END as given in each record, " +
+                "by an amount specified in the argument \"padding\" to look for matching truth record, then " +
+                "tests if the haplotypes by the truth and the evaluation calls are concordant. ",
+        programGroup = StructuralVariantDiscoveryProgramGroup.class)
+public final class EvaluateConcordanceOnNonGenotypedSVVCFs extends AbstractIntervalTreeBasedConcordanceWalker {
+
+    @Argument(doc = "file containing non-canonical chromosome names (e.g chrUn_KI270588v1) in the reference; " +
+            "all chromosome will be considered canonical if not provided",
+            shortName = "alt-tigs",
+            fullName = "non-canonical-contig-names-file",
+            optional = true)
+    private String nonCanonicalChromosomeNamesFile;
+
+    @Advanced
+    @Argument(doc = "indicate evaluation is being performed on non-human calls",
+            shortName = "nonhs",
+            fullName = "non-human",
+            optional = true)
+    private Boolean nonHumanVCF = false;
+
+    @Advanced
+    @Argument(doc = "string value for chromosome Y; takes value of \"Y\" or \"chrY\", or when left unprovided, " +
+            "evaluation is skipped on chromosome Y; applicable only when \"non-human\" is false",
+            shortName = "chry",
+            fullName = "chr-y-string",
+            optional = true)
+    private String chrYString;
+
+    /**
+     * A fixed amount of padding for each VCF record from the eval call set.
+     * Each individual record might be padded (more or less bases) as judged appropriate by the tool itself.
+     */
+    @Argument(doc = "Padding around variants for overlapping",
+            fullName = "padding",
+            optional = true)
+    private Integer fixedPadding = 50;
+
+    @Argument(doc = "prefix for output",
+            shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME,
+            fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME)
+    private String outPrefix;
+
+    @Argument(doc = "Minimum reciprocal overlap fraction for two concordant variants of the type DEL and INV",
+            fullName = "recip-ovp-frac",
+            optional = true)
+    private Float recipOvpFrac = 0.5f;
+
+    @Argument(doc = "Maximum difference in length between two concordant variants or type INS and/or DUP, " +
+            "i.e. the difference in their length cannot exceed this fraction of the shorter of the two",
+            fullName = "max-ins-len-diff-frac",
+            optional = true)
+    private Float insLenDiffFrac = 0.1f;
+
+    @Advanced
+    @Argument(doc = "Minimum mapping quality of the evidence contig mappings for a eval VCF record to be considered;" +
+            " setting to zero turns off this filter",
+            fullName = "min-map-qual",
```

Comment:
Same every-where, why use Wrappers instead of primitives?
Use constants for argument names.